tribe-ext-settings-import-export.php

```diff
@@ -253,76 +233,94 @@ function tribe_sie_process_settings_import() {
 				return;
 			}
 
-			$import_file = $_FILES['import_file']['tmp_name'];
-			$import_filename = $_FILES['import_file']['name'];
-
-			if ( empty( $import_file ) ) {
-				wp_die( __( 'Please upload a file to import.', 'tribe-ext-settings-import-export' ) );
+			/**
+			 * Export actions.
+			 * Process a settings export that generates a .json file of the shop settings.
+			 */
+			if ( ! empty ( $_POST['export'] ) ) {
+				$settings = get_option( 'tribe_events_calendar_options' );
+
+				// TEC - widget_tribe-events-list-widget
+				// PRO - widget_tribe-events-adv-list-widget
+				// PRO - widget_tribe-events-countdown-widget
+				// PRO - widget_tribe-mini-calendar
+				// PRO - widget_tribe-events-venue-widget
+				// PRO - widget_tribe-events-venue-widget
+				// CE  - tribe_community_events_options
+				// CE  - Tribe__Events__Community__Schemaschema_version
+
+				ignore_user_abort( true );
```

Comment:
Not completely sure this is appropriate: this can be useful if we want to keep doing work after the client kills the connection, but in this case I'd imagine there is no point carrying on (because they've killed the connection and we can't actually deliver anything).

Reply:
Just how do you know these things?!? :)
Thanks!